res/database/Default/user/scr0llbaer.xml

```diff
@@ -419,30 +418,75 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="d6242a49-71e0-4997-97d6-3684e5c67d54" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="d6242a49-71e0-4997-97d6-3684e5c67d54" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+			<title>
+				<de>%tower% Dynamit - Das Magazin: %latebreaking%</de>
```

Comment:
I gave chatgpt a go for:
> Erzeuge alternative Titel für folgende Promi-Klatsch und Tratsch-Show mit dem derzeitigen Namen "Dynamit - Das Magazin"

Output:
Explosiv - Die Show mit brisanten News
Puls - Der Ticker für Promi-News
Blitze - Die Sendung mit aktuellen Skandalen
Buzz - Das Magazin mit dem neuesten Klatsch und Tratsch
Feuerwerk - Die Show mit den heißesten Gerüchten
Blitzlicht - Die Sendung mit den neuesten Promi-Storys
Inferno - Die Show mit den explosivsten Enthüllungen
Zündstoff - Das Magazin mit den heißesten Gerüchten aus der Promi-Welt
Bumms - Die Sendung mit den Knallern aus der Promi-Szene
Furore - Das Magazin mit den neuesten Skandalen und Enthüllungen.

> Also give English translations of the suggestions

Output:
Explosive - The show with explosive news
Pulse - The ticker for celebrity news
Flash - The show with current scandals
Buzz - The magazine with the latest gossip and rumors
Firework - The show with the hottest rumors
Spotlight - The show with the latest celebrity stories
Inferno - The show with the most explosive revelations
Fuel - The magazine with the hottest rumors from the celebrity world
Boom - The show with the blasts from the celebrity scene
Furore - The magazine with the latest scandals and revelations.

Maybe this gives some ideas :)

Reply:
We can "verwursten" those eventually but for this I was thinking more along the line that "Dynamit" should be one of  the recognizable brands in the TVTower universe. Hence the idea to change the already existing programme to a "Special" (which doesn't wear-off as fast as the daily show as it documents history).

Reply:
Well, Dynamit would still be among the choices, but with 100 productions, you would probably by the script only once - other names for other players.

Reply:
If I add 10 variations of that, then the "brand recognizability" will be watered down.

There are still some variations so that the "Dynamite" is available for other players too.
In the real world this happens also, sometimes such branded formats are aired by more than one TV station (e.g. "Spiegel TV").

I'd like to use these variations for other things later.

Reply:
ok I added it as a new entry, although with a TODO because it feels it should be more distinct eventually.

Reply:
My suggestion would be to remove the title variables here: just make it "Dynamite - The Magazine".
With the proposed title handling, the result would be one "good" script. For the copy there could be several.

Reply:
It shouldn't be exclusively available to only one player/channel (if that is what removing variables leads to). Kinda like "Spiegel TV", broadcasted on various channels with slight variations, actually very similar.

I'm fine with such slight variations conceptually (just some wordings could be more original in some places).